### ISSUE TITLE
LightGallery creates thousands of event listeners and freezes

### ISSUE THREAD

**Issue by Windowsfreak** [sachinchoolur/lightgallery.js#75]
On galleries with more than 300 images, LightGallery creates way too many (over ten thousand) listeners and then hangs in a loop when trying to close the preview window again.
![image](https://user-images.githubusercontent.com/1639832/31048879-e7182b00-a627-11e7-9ddf-de3fb16cddff.png)

**Comment by sachinchoolur** [comment]
Hey @Windowsfreak,
Could you please provide a jsfiddle which demonstrates the issue. I've tested the gallery with more than 500 images and it works fine. And I don't see any chance of creating thousands of event listeners by lightGallery

**Comment by Windowsfreak** [comment]
Sorry, but by the time I've already switched to a different gallery.

I can still try to give you a vague description to help you reproduce it: Try opening, closing and navigating a few times through the pictures, then close the gallery using the X button in the corner of the page. That was enough to crash smartphones and also the Chrome browser on a desktop.

I used flexgallery for the thumbnail arrangement on the page in combination with lightgallery for the preview window. Flexgallery repositions and resizes the thumnail images but should probably not interfere with lightgallery at all.
